fred2/fred.rc

```diff
@@ -938,15 +938,15 @@ BEGIN
     GROUPBOX        "Default Display",IDC_DISPLAY,107,7,136,56
 END
 
-IDD_SHIP_EDITOR DIALOGEX 0, 0, 321, 422
+IDD_SHIP_EDITOR DIALOGEX 0, 0, 322, 478
 STYLE DS_SETFONT | DS_MODALFRAME | DS_3DLOOK | DS_CENTER | WS_POPUP | WS_CAPTION | WS_SYSMENU
 EXSTYLE WS_EX_NOPARENTNOTIFY | WS_EX_CLIENTEDGE | WS_EX_CONTEXTHELP
-CAPTION "Edit Ship"
+CAPTION " "
```

Comment:
Was this intentional?

Reply:
Nope.  I have no idea how that happened.